Documentation/requirements.txt

```diff
@@ -1,34 +1,35 @@
 alabaster==0.7.12
 Babel==2.9.1
-certifi==2018.10.15
-chardet==3.0.4
-docutils==0.14
-idna==2.7
-imagesize==1.1.0
-Jinja2==2.11.3
-jsonschema==2.6.0
-MarkupSafe==1.1.1
-pyenchant==2.0.0
-Pygments==2.7.4
-pytz==2018.7
-PyYAML==5.4
-recommonmark==0.4.0
-requests==2.25.1
-semver==2.9.0
-six==1.15.0
-snowballstemmer==1.2.1
-Sphinx==1.8.1
-sphinx-autobuild==0.7.1
+certifi==2021.10.8
+chardet==4.0.0
+docutils==0.17
+idna==3.3
+imagesize==1.3.0
+Jinja2==3.0.3
+jsonschema==4.4.0
+# for m2r (dependency to sphinxcontrib-openapi), see https://github.com/miyakogi/m2r/issues/66
+mistune<2.0.0
+MarkupSafe==2.1.0
+myst-parser==0.17.0
+pyenchant==3.2.2
+Pygments==2.11.2
+pytz==2021.3
+PyYAML==6.0
+requests==2.27.1
+semver==2.13.0
+six==1.16.0
+snowballstemmer==2.2.0
+Sphinx==4.5.0
+sphinx-autobuild==2021.3.14
 # forked read the docs themez
-git+https://github.com/cilium/sphinx_rtd_theme.git@v0.7; platform_machine != "aarch64"
-sphinx-rtd-theme==0.2.4; platform_machine == "aarch64"
-sphinxcontrib-httpdomain==1.7.0
-sphinxcontrib-openapi==0.3.2
-sphinxcontrib-spelling==4.2.1
-sphinxcontrib-websupport==1.1.0
-sphinx-tabs==1.1.13
+git+https://github.com/qmonnet/sphinx_rtd_theme.git@wip/rebase
```

Comment:
One more reference to bump before we merge.

Reply:
Absolutely. This was conditioned by the discussion above - Since we agreed on creating a `1.x` branch on that repo, I'll go ahead and do that (if I have the permissions, haven't checked yet). Then I'll come back to this PR and update this line.

Reply:
Now updated to point to a new branch https://github.com/cilium/sphinx_rtd_theme/tree/v1.0. This branch is a rebase of branch `v0.7`, on top of the latest upstream commits. Some clean-up (commit squashing, drop of indentation-only changes, etc.) were done during that rebase, to facilitate potential rebases in the future. Our four latest commits were omitted (because they came after I did that rebase, and the last two are reverts of the former two, so they're all a no-op).

The `master` branch on that repo is now identical to the one in the upstream repository (https://github.com/readthedocs/sphinx_rtd_theme).